ramls/item.json

```diff
@@ -176,6 +176,34 @@
             "type": "boolean",
             "description": "Records the fact that the note should only be displayed for staff",
             "default": false
+          },
+          "source": {
```

Comment:
If these properties are intended to be stored, then the required version of the instance-storage interface should be changed to reflect the expanded storage properties.

Reply:
done